Spawn_tombailey94.js

```diff
@@ -7,37 +10,125 @@ function procCmd(cmd) {
 			clientMessage("You forgot say which mob you want to spawn.");
 		}
 		else if (commandElements[1].toLowerCase() == "chicken") {
-			Level.spawnMob(getPlayerX,getPlayerY,getPlayerZ,10,"mob/chicken.png")
+			Level.spawnMob(getPlayerX,getPlayerY,getPlayerZ,10,"mob/chicken.png");
```

Comment:
Should use EntityRenderType constants (https://github.com/Connor4898/ModPE-Scripts/wiki/Render-types)